index.js

```diff
@@ -90,9 +90,9 @@ let executeHooksWithArgs = (hooks = [], args) => {
             } catch (e) {
                 console.error(e.stack)
                 return resolve(e)
+            } finally {
+                commandIsRunning = _commandIsRunning
```

Comment:
I believe the finally block won't be executed in an error case either so we should have it before the return block as well